src/style/color-palette-extended.scss

```diff
@@ -0,0 +1,306 @@
+@import './mixins';
+
+// To be able to use colors with alpha, they are written in RGB.
+// Comments with HEX codes in front of the values are used as reference only.
+// Example of use:
+// solid: `color: rgb(var(--contrast-100))`
+// transparent: `color: rgba(var(--contrast-100), 0.5)`
+
+:root {
+    // LimeTechnologies' Brand Colors (Do not have dark/light mode variants)
```

Comment:
```suggestion
    // Lime Brand Colors (Does not have dark/light mode variants)
```

Reply:
Based on previous conversations, I'd say `Lime Technologies brand colors`.

Reply:
Err… I'm not sure why I wanted to change from "Do not" to "Does not" here… "does" is singular… but it kinda does sound better… 🤔

Reply:
:zap: